src/risedevtool/src/compose.rs

```diff
@@ -39,7 +39,6 @@ pub struct ComposeService {
     pub depends_on: Vec<String>,
     pub volumes: Vec<String>,
     pub entrypoint: Option<String>,
-    pub environment: BTreeMap<String, String>,
```

Comment:
I think this is actually used in generating `docker-compose.yml`, although `risedev compose` is unmaintained for a while.

Reply:
Can we just remove it, since it's not used by any code (can successfully compile after removing the code), though it's not related to the main purpose of this PR.

Reply:
IIUC it's used to deserialize into a yaml.

Please either don't touch it or remove `compose` completely.

Reply:
Reverted the changes on compose.rs.